new_task/validator.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env pypy3
+#!/usr/bin/env python3
```

Comment:
pypy3 is probably going to validate faster, but I guess it isn't crucial

Reply:
Yes, and at the moment pypy3 is quite broken on archlinux.